src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy

```diff
@@ -29,6 +31,13 @@ class SwitchPairs {
     SwitchPairs nonNeighbouring() {
         return new SwitchPairs(switchPairs.findAll { it.paths.min { it.size() }?.size() > 2})
     }
+    SwitchPairs neighbouring() {
+        return new SwitchPairs(switchPairs.findAll { it.paths.min { it.size() }?.size() == 2})
+    }
+
+    SwitchPairs excludePair(SwitchPair excludePair) {
```

Comment:
Would be better to have an ability to pass list of switch pairs here.

Reply:
Done.